bin/uc

```diff
@@ -45,4 +45,4 @@ quoted_args_str="${quoted_args[*]}"
 
 CLI_JAVA_COMMAND="java -cp $(cat "$CLI_CLASSPATH_FILE") io.unitycatalog.cli.UnityCatalogCli"
 # echo "Running command: $CLI_JAVA_COMMAND"
-eval $CLI_JAVA_COMMAND $quoted_args_str  || exit
+eval "$CLI_JAVA_COMMAND" "$quoted_args_str" || exit
```

Comment:
will this change eliminate the need for all the quote handling above?
and does this work with all shells?

Reply:
this works on bash, sh, and zsh, at least. 

There's probably at least some shell (nushell?) where this doesn't work.

As for eliminating quote handling, maybe? I think it would.

Reply:
I tried by just using the proposed quotes in this PR and not quoting each argument separately. It doesn't work for  macOS + zsh combination.
Given that we already quoting each argument separately do we need to do this overall ?

Reply:
I'm not strongly opinionated either way. I originally created this PR to stop the IntelliJ lints.

Reply:
FWIW, using `"$@"` is the typical way to pass command line arguments to sub-commands and maintain the arguments as-is. It _should_ allow you to remove all the business you are doing to build `quoted_args` which uses `"$@"` itself, so all you really need in for this file is to do `eval $CLI_JAVA_COMMAND "$@"  || exit`, and remove all the code to build `quoted_args`.

Reply:
@abrassel Can you made the changes I suggested earlier? That should be sufficient.

Reply:
sure. i'll have time to get to them this weekend.